stage0/run.go

```diff
@@ -167,6 +167,13 @@ func generatePodManifest(cfg PrepareConfig, dir string) ([]byte, error) {
 		}
 
 		if execOverride := app.Exec; execOverride != "" {
+			// Create a minimal App section if not present
+			if am.App == nil {
+				ra.App = &types.App{
+					User:  "0",
```

Comment:
Would it make sense to get the current UID/GID to support future case where we support to run as a different user?